src/routes.json

```diff
@@ -42,44 +42,44 @@
     {
         "name": "conference-index",
         "pattern": "^/conference/?$",
-        "routeAlias": "/conference(?!/201[4-5])",
+        "routeAlias": "/conference(?!/201[4-7])",
```

Comment:
This pattern will unfortunately break the `2016` conference pages in production, since they'll now be included in the range that's excluded here (i.e. `4-7`).